acceptance-tests/tests/workflows/accountManagementFlow.spec.ts

```diff
@@ -1,36 +1,25 @@
-import { describe, beforeAll, it, expect, afterAll } from 'vitest';
+import { describe, it, expect, afterEach, beforeEach } from 'vitest';
 import { TestState } from '../../lib/TestState';
 import { getInitPromptSequence, ENTER } from '../../lib/prompt';
 
 describe('Account Management Flow', () => {
   let testState: TestState;
   const accountName = 'test-account';
 
-  beforeAll(async () => {
+  beforeEach(async () => {
```

Comment:
Curious about this switch to `beforeEach`. If I'm interpreting this correctly, I think it kind of negates the intent of these "workflow" tests. The idea of the workflow tests is for them to sequentially run each step of the flow the same way that a CLI user would run them. This gives us the opportunity to catch bugs that could disrupt workflows, rather than just testing that each command in isolation works. Typically, devs don't run `hs init` before every account-related command (like `hs accounts list`).

Reply:
Gotcha, I misunderstood the nature of the tests. I can change back.

Reply:
Addressed in https://github.com/HubSpot/hubspot-cli/pull/1123/commits/4d29cfa5d53a3cd02049061caea52a460d5ef11f